devenv/bootstrap.py

```diff
@@ -90,7 +79,45 @@ def main(coderoot: str, argv: Sequence[str] | None = None) -> ExitCode:
 
     os.makedirs(coderoot, exist_ok=True)
 
-    if args.repo == "sentry":
+    if args.repo == "ops":
+        if not os.path.exists(f"{coderoot}/ops"):
+            proc.run(
+                (
+                    "git",
+                    "-C",
+                    coderoot,
+                    "clone",
+                    # TODO: temp
+                    "--single-branch",
+                    "--branch=devenv-poc",
```

Comment:
TODO: get stuff mainlined in a compatible way! Then merge this and release, then bump release version in install-devenv.